scripts/west_commands/twister_cmd.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2022 Google
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import os
+import sys
+
+from west import log
+from west.commands import WestCommand
+
+from zephyr_ext_common import get_twister_path
+
+# Resolve path to twister libs and add imports
+twister_path = get_twister_path()
+os.environ["ZEPHYR_BASE"] = str(twister_path.parent)
```

Comment:
Reminder to address this comment: https://github.com/zephyrproject-rtos/zephyr/pull/50601#discussion_r990503215